benchmarks/datasets.py

```diff
@@ -73,3 +73,18 @@ def _synth_regression_dataset(n_samples=1000, n_features=10000,
         X = sp.csr_matrix(X)
 
     return X, y
+
+
+@M.cache
+def _synth_classification_dataset(n_samples=1000, n_features=10000,
+                                  n_classes=2, dtype=np.float32):
+
```

Comment:
SVC supports sparse input. Can you add a representation parameter, as in `_synth_regression_dataset` to be able to benchmark dense and sparse ?